features/step_definitions/gift_steps.rb

```diff
@@ -1,7 +1,39 @@
-### THEN
+### NOTE: many of these steps rely on the background in gift.feature
 
 Then /^"(.+)" should be notified by email about their gift "(.+)"$/ do |recipient, title|
   step %{1 email should be delivered to "#{recipient}"}
   step %{the email should contain "A gift work has been posted for you"}
   step %{the email should link to the "#{title}" work page}
 end
+
+When /^I have given the work to "(.*?)"/ do |recipient|
+  step %{I give the work to "#{recipient}"}
+  step %{I post the work without preview}
+end
+
+Given(/^I have refused the work/) do
+  step %{I have given the work to "giftee1"}
+  step %{I am logged in as "giftee1" with password "something"}
+  step %{I go to my gifts page}
+  step %{I follow "Refuse Gift"}
+end
+
+When /^I have removed the recipients/ do
+  fill_in("work_recipients", with: "")
+  step %{I post the work without preview}
+end
+
+Then /^"(.*?)" should be listed as a recipient in the form/ do |recipient|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.